packages/tabs/src/elements/TabList.tsx

```diff
@@ -10,7 +10,7 @@ import { StyledTabList } from '../styled';
 import { useTabsContext } from '../utils/useTabsContext';
 
 /**
- * Accepts all `<div>` props
+ * Provides styling to child `Tab` components. Accepts all `<div>` props.
```

Comment:
Hmm, feels a bit limiting to say "Provides styling" as there's also a11y wrapped-up in this component.

Reply:
Nice catch. Maybe: 

`Provides structure and a11y attributes to...`
`Provides structures and necessary a11y to...`
`Provides styling and accessibility attributes to...`

Reply:
I think a generic description would be most suitable since HTML attributes can be applied for various reasons (styling, a11y, general web behavior). For example (very contrived), one could pass `contentEditable` as an attribute.

I think the original description communicates that well.

Reply:
I think my initial thought was that this wrapper is more than just a `div` that accepts all `div` props. It provides a utility to intended child components within it. Does that make sense? Maybe the description I'm proposing still doesn't get that across. 

As a new user, if I ended up here without going through the code to inspect the structure of how it's used in context, I don't really know what it does. So, the hope here is to give a reader some semblance of what it's being used for. I think my goal is a reader could leave the description saying "Oh this is the wrapper component for a list of `Tab` elements."

Reply:
@rossmoody it might be helpful for you to understand why this element exists. It speaks directly to the WAI-ARIA pattern described here: https://www.w3.org/TR/wai-aria-practices/#tabpanel. The link to `tablist` contains helpful text: "A list of tab elements, which are references to tabpanel elements."

Reply:
Yeah! This is the missing context for me, personally. I think it seemed a shame initially (with the absence of any properties to infer use) to say nothing about the component utility except it accepts div props. It seems like this is obvious info from a dev standpoint so I'll revert this to original.

Reply:
Henry was proactive and awesome and helped me get a more appropriate/rounded sense of goals for API takeaways. This type of granular commentary would cause problems. ty ty team